scripts/dev-setup.sh

```
@@ -39,18 +39,43 @@ else
   ok ".venv created"
 else
   ok ".venv already exists"
 fi
 
+# Detect OS-style venv python
+if [ -f "$REPO_ROOT/.venv/Scripts/python.exe" ]; then
+  VENV_PY="$REPO_ROOT/.venv/Scripts/python.exe"
+  VENV_PIP="$REPO_ROOT/.venv/Scripts/pip.exe"
+else
+  VENV_PY="$REPO_ROOT/.venv/bin/python"
+  VENV_PIP="$REPO_ROOT/.venv/bin/pip"
```

Shellcheck (0.11.0)

[warning] VENV_PIP appears unused. Verify use (or export if used externally).

(SC2034)